.github/workflows/main.yml

```diff
@@ -46,19 +46,19 @@ jobs:
       - name: Run PyInstaller
         run: |
           python -m PyInstaller tdmgr32.spec
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v3
         with:
-          name: Windows_tdmgr32
+          name: app
```

Comment:
Why this change? Just curious

Reply:
We need the same path for uploaded artifacts. The name of the apps is not changed.